devops/openshift/database/patroni-deploy.yml

```diff
@@ -186,16 +215,21 @@ parameters:
   description: Maximum amount of memory the container can use.
   displayName: Memory Limit
   value: 512Mi
-- name: BUILD_NAMESPACE
+- name: IMAGE_STREAM_NAMESPACE
   description: |
     The OpenShift Namespace where the patroni and postgresql
     ImageStream resides.
   displayName: ImageStream Namespace
-  required: true
-- name: IMAGE_STREAM_TAG
+  value: bcgov
+- name: IMAGE_NAME
+  description: |
+    The Patroni image stream name
+  value: patroni-postgres
+- name: IMAGE_TAG
   description: |
-    Patroni ImageTag
-  value: patroni:12-latest
+    The image tag used to specify which image you would like deployed.
+    Don't use `latest`.
```

Comment:
The comment is confusing 😄 
"Don't use `latest`" and then we use "12.4-latest".

Reply:
:) Copy paste from the repo sample i got from bcgov. I will remove that line, might be an old comment i guess.